.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+on: [push, pull_request]
```

Comment:
We should specify a branch.
Push = merge pull request
I think statement to test every pull request is correct...